e2e2/test/cases/nvidia/manifests/mpi-job-nccl-test-multi-node.yaml

```diff
@@ -91,11 +91,7 @@ spec:
             resources:
               requests:
                 nvidia.com/gpu: {{.GpuPerNode}}
-                hugepages-2Mi: 5120Mi
```

Comment:
- did we see any failure cases indicates this config has problem? can you show some log?
- maybe for next PR, should we dynamically allocate the memory for different instance types?

Reply:
For 1, I manually run the test are saw the error in the pod status.
For 2, what would happen if we don't dynamically allocate the memory for different instance types?

Reply:
For 2 I don't know the answer but we might need some tuning in the future when we support more instances. I'm ok with removing it to unblock for now